runtime/prelude.jakt

```diff
@@ -19,6 +19,7 @@ extern struct Array<T> {
     function resize(mut this, anon size: usize) throws
     function shrink(mut this, anon size: usize) -> void
     function push(mut this, anon value: T) throws
+    function push_values(mut this, anon other: &Array<T>) throws
```

Comment:
Discard the previous note lol.
Sidenote: Should we call this `extend` or keep it as `push_values`?

Reply:
I wanted to name it `extend` originally but then saw that Array actually already has a push_values function that does kind of the same thing, so I thought this would be the more appropriate name, but I now see that `AK::Vector` uses `extend` as well, so happy to rename it. Should I "rectify" and rename the original `push_values` as well or would it be better to not touch this one as they are expecting different parameters?

Reply:
Let's just keep it `push_values` for now.